chapter2/tests/t6/chapter2-l2-t6-get-file-unique-dirs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# The script return unique directories of all files (including nested directories).
+set -eou pipefail
+
+IFS=$'\n'
+
+if [[ "${#}" -ne 1 ]]; then
+  echo "Usage: [DIR]"
+  exit 1
+fi
+
+dir="${1}"
+
+if [[ ! -e "${dir}" || ! -d "${dir}" ]]; then
+  echo "The '${dir}' does not exist or it is not a directory." >&2
+  exit 1
+fi
+
+#######################################
+# Get files directories (including nested directories).
+# Globals:
+#   Directory.
+# Outputs:
+#   Writes directories into stdout.
+#######################################
+function get_files_directories {
+  for file in $(find "${dir}" -type f); do
+    echo "${file%/*}"
+  done
+}
+
+get_files_directories | sort -u
+
```

Comment:
Too many empty lines at the end, should be one

Reply:
![image](https://user-images.githubusercontent.com/8140942/232480221-7e4ab909-c4fa-4a69-936e-ef19d2263c52.png)
Do you want me to remove end EOL? Or space between function body and `get_files_directories` call? Pls advise because I see only one new line at the end.

Reply:
Fixed it.